Atomic/containers.py

```diff
@@ -187,9 +187,9 @@ def _check_filters():
                 return True
             for f in self.args.filters:
                 _filter, _ = f.split('=', 1)
-                if _filter not in [x for x in self.FILTER_KEYWORDS]:
+                if _filter not in self.FILTER_KEYWORDS.keys():
                     raise ValueError("The filter {} is not valid.  "
-                                     "Please choose from {}".format(_filter, [x for x in self.FILTER_KEYWORDS]))
+                                     "Please choose from {}".format(_filter, self.FILTER_KEYWORDS.keys()))
```

Comment:
The thing about using keys() directly is that it will display: 
`Please choose from dict_keys(['state', 'container', 'command', 'backend', 'runtime', 'image', 'created'])`
instead of:
`Please choose from ['repo', 'tag', 'id', 'created', 'size', 'type', 'dangling']`
Which looks a little worse. Not a huge deal.